src/ui/server/controllers/root/thankyou/index.ts

```diff
@@ -0,0 +1,20 @@
+import { Request, Response } from '../../../../types';
+
+/**
+ * Handles the GET request for the thank you page.
+ *
+ * This function sets the response type to plain text and writes a thank you message
+ * with a placeholder for the date, name, and description.
+ *
+ * @param req - The request object.
+ * @param res - The response object.
+ */
```

Comment:
Can we follow the same req/res documentation pattern we have in other places please to be consistent (aligned these not long ago 😛)

Same with the other new controller

e.g https://github.com/UK-Export-Finance/exip/blob/main/src/ui/server/controllers/insurance/all-sections/index.ts#L13